README.md

```diff
@@ -14,8 +14,8 @@ This repository itself can be cloned and edited to your needs. The skeleton prep
 * **Tests**: created with [Tape](https://github.com/substack/tape) in the `test/` directory. Travis CI file is prepared to build and test your project on every push.
 * **Documentation**: use this README as a template and customize for your own project. Also, this skeleton uses [documentation.js](http://documentation.js.org/) to generate API documentation from JSDOC comments in the `.cpp` files. Docs are located in `API.md`.
 * **Build system**: [node-pre-gyp](https://github.com/mapbox/node-pre-gyp) generates binaries with the proper system architecture flags
-* **Publishing**: Structured as a node module with a `package.json` that can be deployed to NPM's registry.
-* **Learning resources**: Read the detailed inline comments within the example code to learn exactly what is happening behind the scenes. Also, check out the [extended tour](./docs/extended-tour.md) to learn more about Node/C++ Addon development, builds, and more specifics about the configuration of this skeleton.
+* **[Publishing](./docs/publishing-binaries.md)**: Structured as a node module with a `package.json` that can be deployed to NPM's registry.
```

Comment:
@GretaCB - did you forget to commit `./docs/publishing-binaries.md`? (I don't see it in this branch)